site/pom.xml

```diff
@@ -12,6 +12,10 @@
 		<version>0.8.0-SNAPSHOT</version>
 	</parent>
 
+	<properties>
+		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}/all/repo/</publishingPath>
```

Comment:
Not technically true (case conventions do not define functionality, only stylistic preference), but I agree that renaming the job to "reddeer_master" or "jbosstools-reddeer_master" would be more consistent w/ the rest of the JBT projects.